### ISSUE TITLE
Support for variable length arrays

### ISSUE THREAD

**Issue by Stewori** [libav/c99-to-c89#22]
```
str1 = whatever;
if (strlen(str1) == 0) return whatever;
char str2[strlen(str1)+ 1];
strcpy(str2, str1);
whatever
```

becomes something like

```
str1 = whatever;
if (strlen(str1) == 0) return whatever;
{ char str2[strlen(str1)+ 1];
strcpy(str2, str1);
whatever
}
```

It could be done near-properly by using alloca, i.e. _alloca (for MSVC):

```
str1 = whatever;
if (strlen(str1) == 0) return whatever;
{ char* str2 = _alloca(strlen(str1) + 1);
strcpy(str2, str1);
whatever
}
```

In case it was a concious decision not to Support VLA, then this should be mentioned in the readme.
In general it would be helpful if known limits of c99-to-c89 were documented, at least roughly.

That said, it is still a more than helpful tool. Thanks for creating it!

**Comment by rbultje** [comment]
I don't think it was conscious, we implemented what we needed to get ffmpeg to compile and then it was "good enough" for us. I think we'd be happy to merge patches to support other (non-ffmpeg) features, but realistically a lot of people have moved on now that recent versions of MSVC semi-properly support C99...
